media/js/firefox/new/experiment-win64-funnelcakes-scene2.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    // update dl link for windows to point to win64
+    // same selector from canonical scene2.js
+    var $platformLink = $('#download-button-wrapper-desktop .download-list li:visible .download-link');
```

Comment:
Hm, actually, I think your suggestion is better, as if somehow the visible `li` is not the Windows one, and the user somehow gets the Windows download URL, it will be incorrect (having `os=win` instead of `os=win64`). Will update.